modules/Microsoft.Network/expressRouteCircuits/readme.md

```diff
@@ -16,19 +16,17 @@ This template deploys an express route circuit.
 | `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
 | `Microsoft.Authorization/roleAssignments` | [2020-10-01-preview](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2020-10-01-preview/roleAssignments) |
 | `Microsoft.Insights/diagnosticSettings` | [2021-05-01-preview](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Insights/2021-05-01-preview/diagnosticSettings) |
-| `Microsoft.Network/expressRouteCircuits` | [2021-05-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-05-01/expressRouteCircuits) |
+| `Microsoft.Network/expressRouteCircuits` | [2022-01-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/expressRouteCircuits) |
```

Comment:
Please update to latest version documented in ARR so that the link is mapped to the specific version used and not to generic latest (which can change over time)

Reply:
Updated to older API version